articles/load-balancer/load-balancer-distribution-mode.md

```diff
@@ -44,12 +44,20 @@ Another use case scenario is media upload. The data upload happens through UDP,
 
 ## Configure source IP affinity settings
 
-For virtual machines, use Azure PowerShell to change the timeout settings. Add an Azure endpoint to a virtual machine and configure the load balancer distribution mode:
+For classic virtual machines, use Azure PowerShell to change the distribution settings. Add an Azure endpoint to a virtual machine and configure the load balancer distribution mode:
 
 ```powershell
 Get-AzureVM -ServiceName mySvc -Name MyVM1 | Add-AzureEndpoint -Name HttpIn -Protocol TCP -PublicPort 80 -LocalPort 8080 –LoadBalancerDistribution sourceIP | Update-AzureVM
 ```
 
+For virual machines deployed with Resource Manager, use PowerShell to change the load balancer distribution settings on the load balancing rule of te load balancer.  This would update the distribution mode:
+
+```powershell
+$lb = Get-AzureRmLoadBalancer -Name MyLb -ResourceGroupName MyLbRg
+$lb.LoadBalancingRules[0].LoadDistribution = 'sourceIp'
+Set-AzureRmLoadBalancer -LoadBalancer $lb
+```
```

Comment:
This changes the distribution mode of an existing rule, whereas the document describes creation of a new rule. We need to cover both for ARM

Reply:
Needs Az CLI 2.0 instructions as well